creator/analyses/file_types.py

```diff
@@ -41,6 +41,28 @@
         ],
         "template": "participant_config.py",
     },
+    "PTC": {
+        "name": "Participant Conditions",
+        "required_columns": [
+            "Participant ID",
+            "Age at Onset",
+            "Age at Onset Units",
+            "Condition Name",
+            "Verification Status",
+        ],
+        "template": "participant_conditions_config.py",
+    },
+    "PTO": {
+        "name": "Participant Observations",
+        "required_columns": [
+            "Participant ID",
+            "Age at Onset",
+            "Age at Onset Units",
+            "Observation Name",
+            "Verification Status",
+        ],
+        "template": "participant_observations_config.py",
+    },
```

Comment:
Per my comment on notion, these conditions and observations forms don't seem to address Kids First needs for diagnosis/phenotype identification.